net-misc/seafile-client/seafile-client-6.0.4-r1.ebuild

```diff
@@ -43,6 +45,7 @@ src_configure() {
 	export QT_SELECT=qt5
```

Comment:
Thanks, you two!
I've added a patch that fixes the build system in #4497.
Since the project doesn't use qmake, I suppose this is the more correct way.
(Compilation only works with Qt5 - but if Qt4 is installed, there is one test for qmake version that then gives the result for Qt4...)